src/test_l1_plat_power.c

```diff
@@ -115,11 +115,11 @@ void test_l1_plat_power_positive_PLAT_INIT (void)
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID); 
     pmStatus_t ret;
 
-    // Variation/Step 01: Call PLAT_INIT() - open interface
+    // Step 01: Call PLAT_INIT() - open interface
     ret = PLAT_INIT();
     UT_ASSERT_EQUAL_FATAL(ret, PWRMGR_SUCCESS); // Ensure the returned value is PWRMGR_SUCCESS
```

Comment:
comment added like this can be removed in whole file

Reply:
Done